engine/storage/snapshot/test/resources/db.properties

```diff
@@ -46,6 +47,8 @@ db.cloud.url.params=prepStmtCacheSize=517&cachePrepStmts=true&prepStmtCacheSqlLi
 db.usage.username=cloud
 db.usage.password=cloud
 db.usage.host=localhost
+# It's not guaranteed that using a different DB provider than the one from the regular cloud DB will work
```

Comment:
Yeah, it should work, it's just a matter of using a different driver, but unfortunately I'm not able to test such a setup :)